docs/fsharp/tutorials/getting-started/getting-started-visual-studio.md

```diff
@@ -5,168 +5,147 @@ keywords: visual f#, f#, functional programming
 
-Visual Studio 2010 includes a new programming language, F#. F# is a multiparadigm language that supports functional programming in addition to traditional object-oriented programming and .NET concepts. The following examples introduce some of its features and syntax. The examples show how to declare simple variables, to write and test functions, to create tuples and lists, and to define and use a class.
+## Installing the Visual F# Tools
 
-Note: General Settings
+After Visual Studio is finished initializing, select **Custom** as shown here:
 
+![](../../media/vs2015-install-1.png)
 
-### To create a new console application
+Select the Visual F# Tools under Programming Languages here:
 
-1. On the **File** menu, point to **New**, and then click **Project**.
-<br />
-
-2. If you cannot see Visual F# in the **Templates Categories** pane, click **Other Languages**, and then click **Visual F#**. The **Templates** pane in the center lists the F# templates.
-<br />
+![](../../media/vs2015-install-2.png)
 
-3. Look at the top of the **Templates** pane to make sure that **.NET Framework 4** appears in the **Target Framework** box.
-<br />
+Feel free to customize your installation further, and then continue with the installation.  After a while, Visual Studio will complete installation and you can create an F# project!
 
-4. Click **F# Application** in the list of templates.
-<br />
+## Creating a Console Application
 
-5. Type a name for your project in the **Name** field.
-<br />
+One of the most basic projects in Visual Studio is the Console Application.  Here's how to do it.  Once Visual Studio is open:
 
-6. Click **OK**.
-<br />  The new project appears in **Solution Explorer**.
-<br />
+1. On the **File** menu, point to **New**, and then click **Project**.
 
+![](../../media/vs2015-install-3.png)
 
-### To use the let keyword to declare and use identifiers
+2.  In the New Project dialog, under **Templates**, you should see ***Visual F#**.  Click this to bring up the F# templates.
 
-1. Copy and paste the following code into **Program.fs**. You are binding each identifier, **anInt**, **aString**, and **anIntSquared**, to a value.
+![](../../media/vs2015-install-4.png)
 
-[!code-fsharp[Main](../../../../samples/snippets/fsharp/contour/snippet1.fs)]
-        
->[!NOTE]
-If you cannot see the code in Classic view, make sure that the **Language Filter** in the header below the topic title is set to include F#.
+3. Click the **Okay** button to create the F# project!  You should see something like this under **Solution Explorer**:
 
-### To see results in the F# Interactive window
+![](../../media/vs2015-install-5.png)
 
-1. Select the **let** expressions in the previous procedure.
-<br />
+>[!NOTE] You might have a different name for your solution and project.  This is okay!
```

Comment:
right format for this is
```
> [!NOTE] 
> You might have a different name for your solution and project.  This is okay!
```
but I don't know if we really need to tell readers this.